Cargo.toml

```diff
@@ -205,12 +205,49 @@ rest_pat_in_fully_bound_structs = "warn"
 
 [profile.release]
 strip = true
-lto = true
+lto = "fat"
```

Comment:
[`"fat"` and `true` are the same](https://doc.rust-lang.org/cargo/reference/profiles.html#lto), but `fat` is more descriptive. The boolean value is a holdover from the early days when `lto` was a strict binary option.